clients/drcachesim/tracer/raw2trace.cpp

```diff
@@ -47,7 +47,7 @@
 #include "utils.h"
```

Comment:
Could we also add a test that ensures that is_maybe_blocking_syscall does the right thing, and make sure it runs in the a64-on-x86 workflow too?

Reply:
That could be done in raw2trace_unit_tests.cpp I guess where you can add a public API to raw2trace_test_t  that calls the protected is_maybe_blocking_syscall.